jumpto/src/org/netbeans/modules/jumpto/file/FileSearchAction.java

```diff
@@ -1,47 +1,20 @@
-/*
- * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
- *
- * Copyright 1997-2010 Oracle and/or its affiliates. All rights reserved.
- *
- * Oracle and Java are registered trademarks of Oracle and/or its affiliates.
- * Other names may be trademarks of their respective owners.
- *
- * The contents of this file are subject to the terms of either the GNU
- * General Public License Version 2 only ("GPL") or the Common
- * Development and Distribution License("CDDL") (collectively, the
- * "License"). You may not use this file except in compliance with the
- * License. You can obtain a copy of the License at
- * http://www.netbeans.org/cddl-gplv2.html
- * or nbbuild/licenses/CDDL-GPL-2-CP. See the License for the
- * specific language governing permissions and limitations under the
- * License.  When distributing the software, include this License Header
- * Notice in each file and include the License file at
- * nbbuild/licenses/CDDL-GPL-2-CP.  Oracle designates this
- * particular file as subject to the "Classpath" exception as provided
- * by Oracle in the GPL Version 2 section of the License file that
- * accompanied this code. If applicable, add the following below the
- * License Header, with the fields enclosed by brackets [] replaced by
- * your own identifying information:
- * "Portions Copyrighted [year] [name of copyright owner]"
- *
- * Contributor(s):
- *
- * The Original Software is the Accelerators module.
- * The Initial Developer of the Original Software is Andrei Badea.
- * Portions Copyright 2005-2006 Andrei Badea.
```

Comment:
Hm, don't change this. We have to ask Andrei Badea for permission.

Reply:
Ok. I will put this four lines back.

Reply:
I must put back this 5 lines in the license header. OK?
* Contributor(s):
*
* The Original Software is the Accelerators module.
* The Initial Developer of the Original Software is Andrei Badea.
* Portions Copyright 2005-2006 Andrei Badea.

Reply:
No, revert the change on the whole file until we figure out what to do with it on the dev@ mailing list.